iothub/service/src/Utilities/NewtonsoftJsonETagConverter.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure;
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.Devices
+{
+    /// <summary>
+    /// A <see cref="JsonConverter"/> implementation for <see cref="ETag"/>.
+    /// </summary>
+    internal class NewtonsoftJsonETagConverter : JsonConverter
+    {
+        /// <inheritdoc/>
+        public override bool CanConvert(Type objectType) => objectType == typeof(ETag) || objectType == typeof(ETag?);
+
+#nullable enable
```

Comment:
We don't use nullable in our projects, so you can remove these directives.

Reply:
Removing these lines breaks the implementation here though. This method is an override of a method that returns a ```object?``` type.